src/main/java/org/example/ngevaticketmanagerspring/user/UserController.java

```diff
@@ -0,0 +1,72 @@
+package org.example.ngevaticketmanagerspring.user;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
```

Comment:
```suggestion

@RequiredArgsConstructor
```

@Autowired wird nicht mehr empfohlen. Zusätzlich kann man sich den Konstruktor für den Service komplett sparen dank @RequiredArgsConstructor
https://medium.com/@dulanjayasandaruwan1998/spring-doesnt-recommend-autowired-anymore-05fc05309dad